README.rst

```diff
@@ -588,6 +588,31 @@ which is indexed in degrees, the 'distance' will be interpreted as degrees. Set
 to 'True' in order to convert an input distance in meters to degrees. This conversion is approximate, and the errors
 at latitudes > 60 degrees are > 25%.
 
+DistanceToPointOrderingFilter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Provides a ``DistanceToPointOrderingFilter``, which is a subclass of DRF
```

Comment:
here we should mention (in bold) that this feature is available only on Django > 3.0, can you add it please?